roles/confluent.variables/vars/main.yml

```
@@ -65,15 +66,16 @@ zookeeper_final_properties: "{{ zookeeper_combined_properties | combine(zookeepe
 
 
 #### Kafka Broker Variables ####
-kafka_broker_service_name: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka'}}"
-kafka_broker_main_package: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka-2.12'}}"
-kafka_broker_packages:
+kafka_broker_service_name: "{{ 'confluent-server' if confluent_server_enabled|bool and not confluent_community|bool else 'confluent-kafka'}}"
+kafka_broker_main_package: "{{ 'confluent-server' if confluent_server_enabled|bool and not confluent_community|bool else 'confluent-kafka-2.12'}}"
```

```suggestion
kafka_broker_service_name: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka'}}"
kafka_broker_main_package: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka-2.12'}}"
```

Since confluent_server_enabled should already be set correctly do you need the and condition?